apps/www/_blog/2023-04-10-o11y-day.mdx

```diff
@@ -0,0 +1,201 @@
+
+The initial request is fast because Logflare also performs setup (such as partitioning) on our BigQuery tables appropriately. Subsequent requests made are extremely fast as they get cached in-memory.
+
+The best part is that all these knobs can be tweaked based on the use case. If we have a hard realtime requirement, we can completely disable caching, or drop the proactive caching to update on a per-second basis.
+
+## The Self-hosted Challenge
+
+In order to execute the license change, we had to get rid of all closed-source dependencies. Prior to our efforts, Logflare relied on the closed source [General SQL Parser](https://www.sqlparser.com/) under a business licenses. However, it would not be an acceptable compromise under the Apache License.
+
+This resulted in an effort to switch Logflare over to an open source alternative, and we settled on the rust-based [sqlparser-rs](https://github.com/sqlparser-rs/sqlparser-rs) library, and also ended up helping to contribute a [few fixes](https://github.com/sqlparser-rs/sqlparser-rs/pulls?q=is%3Apr+is%3Amerged+author%3AZiinc) upstream for the BigQuery dialect.
+
+Besides the parser, a big part of getting Logflare working with Supabase locally was transforming the multi-tenant architecture into something that was self-hosting friendly and could be easily configurable. We also moved towards environment variable based configuration instead of compile-time configurations, and also exposed the Endpoints configurations necessary for Supabase Logs.
+
+## What’s Next?
+
+To further integrate Logflare into the Supabase platform, we are building out 2 main areas: Management API, Multiple Backends.
+
+### Management API
+
+The Management API allows users to programmatically interact with Logflare server and manage their Logflare account and resources. This feature will be available for both Logflare customers and self-hosted users.
+
+You can check out the preview of our OpenAPI spec here: [https://logflare.app/swaggerui](https://logflare.app/swaggerui)
+
+![Swagger UI](/images/blog/lw7-o11y/swagger.png)
+
+Not only that, we intend to expose user account provisioning to select partners. Soon, you’ll be able to apply to become a Logflare Partner and provision Logflare accounts with the Partner API. Perfect if you want to resell a log analytics service from within your platform.
+
+Contact us at growth@supabase.com to get in early on that waitlist.
+
+## Multiple Backends
+
+We understand that many of our customers are concerned about relying on closed-source products. Although Logflare currently supports a BigQuery backend, we hope to add support for other analytics optimized databases like Clickhouse. Beyond that, we can also support pushing data to other web services, allowing Logflare to fit into any data pipeline.
+This would benefit our Supabase CLI integration, such that once Postgres support is available, Logflare would be able to integrate seamlessly out-of-the-box without the BigQuery requirement.
```

Comment:
```suggestion
This will benefit the Supabase CLI: once Postgres support is available, Logflare will be able to integrate seamlessly, without the BigQuery requirement.
```